src/web/lib/components/PresetThemeSelector/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import BrowserPreview from '../BrowserPreview';
+
+import { presetColors } from '../../../../lib/constants';
```

Comment:
Might be better to get the presetColors as props.

Reply:
Yeah, though I think this is okay for now. Maybe refactor once we want to expand the list of themes